views/scss/qti/_inline-choice.scss

```diff
@@ -27,8 +21,13 @@ select.qti-inlineChoiceInteraction,
     font-weight: normal !important;
 }
 
-// shadow like focused element
+
 .qti-inlineChoiceInteraction {
+    &.select2-container {
+        max-inline-size: 100%;
```

Comment:
If option label is huge. Should be fine to set this for horizontal too, not only for vertical. 

Without this, test-taker mode example:
![Screenshot_78](https://github.com/user-attachments/assets/4d1d13c4-d02e-4793-b532-dcc0b003da22)


With this, review mode example:
![Screenshot_79](https://github.com/user-attachments/assets/e2fe3ca5-b3f7-4097-ab86-4d9176f3c9a9)